glossary.md

```diff
@@ -14,15 +11,11 @@ This page contains a selection of key terms important to understanding the struc
 
 - **[Statistical variable](https://datacommons.org/browser/StatisticalVariable)**
 
-Any type of metric, statistic, or measure that can be measured at a place and time. Examples include median income, median income of females, number of high school graduates, unemployment rate, or prevalence of diabetes. For more information, see ["Intro to `StatisticalVariable` and `StatVarObservation`"](/contributing/background/representing_statistics.html).
+Any type of metric, statistic, or measure that can be measured at a place and time. Examples include median income, median income of females, number of high school graduates, unemployment rate, or prevalence of diabetes.
```

Comment:
Would be nice to link to some examples in the browser, and to link to the complete list of statistical variables.

Reply:
I changed some of the examples and also linked them. I added a link to the list of statistical variables, but it feels somewhat awkward--let me know what you think!